.github/workflows/api-dotnetcore.yml

```diff
@@ -2,48 +2,46 @@ name: API (.NET Core)
 
 on:
   push:
-    branches: [ master, dev ]
+    branches: [master, dev]
   pull_request:
-    branches: [ master, dev ]
+    branches: [master, dev]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
     env:
       working-directory: ./backend
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Setup .NET Core
-      uses: actions/setup-dotnet@v1
-      with:
-        dotnet-version: 3.1.101
-    - name: Install dependencies
-      run: dotnet restore
-      working-directory: ${{env.working-directory}}
-    - name: Build
-      run: dotnet build --configuration Release --no-restore
-      working-directory: ${{env.working-directory}}
+      - uses: actions/checkout@v2
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 3.1.101
+      - name: Install dependencies
+        run: dotnet restore
+        working-directory: ${{env.working-directory}}
+      - name: Build
+        run: dotnet build --configuration Release --no-restore
+        working-directory: ${{env.working-directory}}
 
   test:
-
     runs-on: ubuntu-latest
     env:
       working-directory: ./backend
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Setup .NET Core
-      uses: actions/setup-dotnet@v1
-      with:
-        dotnet-version: 3.1.101
-    - name: Install dependencies
-      run: dotnet restore
-      working-directory: ${{env.working-directory}}
-    - name: Build
-      run: dotnet build --configuration Release --no-restore
-      working-directory: ${{env.working-directory}}
-    - name: Test
-      run: dotnet test --no-restore --verbosity normal
-      working-directory: ${{env.working-directory}}
+      - uses: actions/checkout@v2
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 3.1.101
+      - name: Install dependencies
+        run: dotnet restore
+        working-directory: ${{env.working-directory}}
+      - name: Build
+        run: dotnet build --configuration Release --no-restore
+        working-directory: ${{env.working-directory}}
+      - name: Test
+        run: dotnet test --no-restore --verbosity normal
+        working-directory: ${{env.working-directory}}
```

Comment:
I'd be tempted to bundle "dotnet test" with the build action above as we are building the API twice. Just a suggestion, the actions seems to run fast anyways

Reply:
I think you’re right.  I separated them originally so we could more clearly identify what failed but I don’t think that is necessary